src/main/scala/draughts/Pos.scala

```diff
@@ -44,6 +45,16 @@ sealed case class Pos100 private (x: Int, y: Int) extends PosMotion(5 * (y - 1)
   lazy val moveLeft  = movesHorizontal.get(fieldNumber).map(_(0)).filter(_ > 0) flatMap posAt
   lazy val moveRight = movesHorizontal.get(fieldNumber).map(_(1)).filter(_ > 0) flatMap posAt
 
+  def getDistFromSide(): Int = {
+    fieldNumber % 10 match {
```

Comment:
same applies below: 
there are no inputs so you dont need the brackets, you also dont need the outside {} as you are doing the calculation in one go. This could then be a val or a def, i dont mind either, it seems that def is more standard practise in sg, therefore the L18 would be
`def getDistFromSide: Int`